src/utility.jl

```diff
@@ -127,6 +127,41 @@ function _replace_sqr(str::AbstractString)
     return str
 end
 
+function _replace_POWER(str::AbstractString)
+    pVal = 0.0
+    for i in 1:length(str)-6
+        if str[i:(i+5)] == "POWER("
+            warp = 0
```

Comment:
I would set `warp` to 1 (as the bracket is open) and the check `if warp == 0` in l.144 makes it more readable, I think,

Reply:
Should it be for the case of POWER(...(...(...))) => INIT=0(, 1(, 2(, 1), 0), -1)?

Reply:
I think it should be `(` => +1 and `)` => -1 so if the same amount is open as closed it should be 0.